src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -34,6 +36,7 @@ const MAX_RESULT_HISTORY_LENGTH = 5;
     styleUrls: ['../course-overview.scss'],
 })
 export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
+    [x: string]: any;
```

Comment:
What is this needed for? Doesn't it basically negate type checking for the whole component by accepting any kind of attribute access?

Reply:
i removed this unnecessary string